src/main/resources/templates/fragments/head.html

```diff
@@ -151,4 +151,6 @@
 
     <script type="text/javascript" src="../js/intro.min.js"></script>
 
+    <link rel="stylesheet" href="../css/tailwind.css">
```

Comment:
I think we have to add this in our project? can we use CDN here instead of keeping in our local?

Reply:
its better to use a local version 
it is same across all the lib we use in the project

Reply:
currently tailwind.css is not added in this pr

Reply:
Got it
We need to build the tailwind.css using the command
`npm install tailwindcss --save-dev`
`npm run build:css` which will automatically build the tailwind.css for our project optimally
The tailwind functionality is working fine 
Add nodemodules inside .gitignore file to avoid nodemodules pushed into project mistakenly
everything looks good to me